spacy_loggers/lookup.py

```diff
@@ -0,0 +1,26 @@
+"""
+A utility logger that looks up specific statistics and prints them to stdout.
+"""
+from typing import Dict, Any, Optional, IO
+import sys
+
+from spacy import Language
+from .util import dict_to_dot, LoggerT
+
+
+def lookup_logger_v1(substring: str) -> LoggerT:
```

Comment:
Could this be a `pattern` instead, so we actually support regex-style matching to the metric names? If we want to make this more generic, doing this right now in v1 might make sense.

Reply:
Good point; will do.